src/renderer/common/components/dialog/publicationInfos/publicationInfoContent.tsx

```diff
@@ -29,6 +30,41 @@ export interface IProps {
     onClikLinkCb?: (tag: IOpdsBaseLinkView) => () => void | undefined;
 }
 
+const Duration = (props: {
+    duration: ITimeDuration;
+    __: I18nTyped;
+}) => {
+
+    const { duration, __ } = props;
+
+    let sentence = "";
+    sentence += duration?.days ? `${duration.days} ${__("publication.duration.days")} ` : "";
+    sentence += duration?.hours ? `${duration.hours} ${__("publication.duration.hours")} ` : "";
+    sentence += duration?.minutes ? `${duration.minutes} ${__("publication.duration.minutes")} ` : "";
+    sentence += sentence ? `${__("publication.duration.and")} ` : "";
+    sentence += duration?.seconds ? `${duration.seconds} ${__("publication.duration.seconds")}` : "";
+
+    sentence = sentence.trim();
+
+    const isDefined = sentence.length > 0;
+
+    return (isDefined ?
```

Comment:
Code consistency: at line `44` there is a ternary expression which checks for "falsy" `sentence` (i.e. empty string) `sentence += sentence ? `${__("publication.duration.and")} ` : "";` ... so I would suggest removing `isDefined` and simply replace with `return (sentence ? ...`.

Reply:
https://github.com/readium/readium-desktop/pull/973/commits/4f30f388a60e72c529070853f1cd62b2dbd68f4d